sim_pipeline/Sources/source_base.py

```diff
@@ -1,7 +1,16 @@
+<<<<<<< HEAD
+from abc import ABC, abstractmethod
+
+class SourceBase(ABC):
+    """
+    Base class with functions all source classes must have to be able to render populations
+    """
+=======
```

Comment:
it looks like you have accepted both versions and also the patterns of the merge conflict '======' and '>>>>>>/

Reply:
Ah, got it! I missed a couple of those fixes, will re-commit!